### ISSUE TITLE
Question on ticketmaster.sg smart queue

### ISSUE THREAD

**Issue by brytjy** [max32002/tixcraft_bot#167]
Would it work if there’s a smart queue implemented for ticketmaster.sg?

**Comment by ganhj99** [comment]
No, this bot will not bypass smart queue/queue-it, but the flow/process after queue will work.
